webpack/config/index.js

```diff
@@ -3,6 +3,17 @@ const path = require('path');
 const staticAssetsDirectory = './website/static/.'; // The folder where static files (not processed) live
 const prodEnv = require('./prod.env');
 const devEnv = require('./dev.env');
+const nconf = require('nconf');
+
+const PATH_TO_CONFIG = path.join(path.resolve(__dirname, '../../config.json'));
+let configFile = PATH_TO_CONFIG;
+
+nconf
```

Comment:
this looks good but we have almost the same code in website/server/libs/setupNconf, can we reuse it?